CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed a bug where union types with inline schema member types would fail to generate #1270
 - Fixed a bug where referenced types with no titles would fail to generate #1271
 - Fixed a bug where the generator would introduce unnecessary union types for nullables. #990
+- TypeScript adding index exporting models to fix #870.
```

Comment:
Can you move this into the unreleased section please?

Reply:
Done. Please check now.